src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldHaveCorrectSuffixTests.cs

```diff
@@ -1380,6 +1380,261 @@ Inherits Dictionary(Of String, String)
             }.RunAsync();
         }
 
+        [Fact, WorkItem(1818, "https://github.com/dotnet/roslyn-analyzers/issues/1818")]
+        public async Task CA1710_AllowEmptySuffix()
+        {
+            var editorConfigText = @"dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader->{}";
```

Comment:
Can you make this a Theory based test and also verify the behavior for different use of whitespaces, say `T:System.Data.IDataReader -> {}` and `T:System.Data.IDataReader-> { } `?